src/app/core/models/business.model.ts

```diff
@@ -46,64 +49,66 @@ export enum DAYS_WEEK {
   SUNDAY = 'SUNDAY'
 }
 
-export function values() {
-  return Object.keys(PAYMENT_TYPE_BUSINESS).filter(
-    (type) => isNaN(type as any) && type !== 'values'
-  );
-}
+// export function businessPaymentTypesValues() {
+//   return Object.keys(BUSINESS_PAYMENT_TYPES).filter(
+//     (type) => isNaN(type as any) && type !== 'values'
+//   );
+// }
 
 export class Category {
   id?: number;
   name: string;
-  businessId?: number;
 }
 
 export class BusinessHour {
   id?: number;
   businessId: number;
-  day: DAYS_WEEK;
+  day: BUSINESS_DAYS_WEEK;
   closingTime: string;
   openingTime: string;
 }
 
 export class Business {
   protected id?: number;
-  protected phoneId?: number;
+  protected phoneId?: number[];
```

Comment:
tu devrais appeler ca phones
```suggestion
  protected phones?: number[];
```

Reply:
ca va dépendre de ce que tu m'envoie depuis l'API.

Reply:
ca va être une liste cest sur